docs/genetic.md

```diff
@@ -338,6 +338,90 @@ size output of {py:func}`sim_trait`:
 
 :::{note}
 The effect size output of {py:func}`sim_genetic` is used in simulating
-the genetic values, and not the `trait_df` input in the frequency dependent
-model.
+the genetic values, as it represents the effect sizes after the frequency
+dependence architecture is accounted.
 :::
+
+## Example: Infinitesimal Model Simulation
```

Comment:
I advise agains using the term `infinitesimal`. This term was initially used by Fisher in his 1918/1919 paper where he motivated Gaussian distribution by saying that there are infinitely many causal loci with infinitely small effects, hence the `infinitesimal` model. Then for decades people were just simulating quantitative genetic studies with only Gaussian distribution without reference to genome. And for many, this is what `infinitesimal` simulation refers to. 1000 loci is of course a lot and resulting distributions look very much Gaussian, but its not `infinitesimal` as Fisher has coined it. Just my 2 pennies.

Reply:
Thanks a lot @gregorgorjanc !